src/main/webapp/profileScript.js

```diff
@@ -22,18 +28,13 @@ function loginStatusDisplay() {
       const personalProfileButton = document.createElement('button');
       personalProfileButton.innerText = 'My Profile';
       personalProfileButton.addEventListener('click', () => {
-        window.location.href = 'profile.html';
+        visitProfile(json['id']);
       });
       userNavbarSection.appendChild(logoutButton);
       userNavbarSection.appendChild(personalProfileButton);
-
-      // Add the user's name to the profile info.
-      const profileName = document.createElement('h1');
-      profileName.innerText = json['name'];
-      userInfo.appendChild(profileName);
+    
+    // If the user is logged in, add a login button to the navbar.
```

Comment:
Nit: I think you meant "if the user is **not** logged in" here

Reply:
Oops yeah, TY.